src/js/profile/mobile/widget/GridView.js

```diff
@@ -622,6 +631,35 @@
 				}
 			};
 
+			/**
+			 * Set checkbox initial state
+			 * @method _initCheckboxState
+			 * @protected
+			 * @member ns.widget.mobile.GridView
+			 */
+			prototype._initCheckboxState = function (element) {
+				var checkboxNodeList = element.querySelectorAll("input[type=checkbox]"),
+					i = 0,
+					self = this;
+
+				for (i = 0; i < checkboxNodeList.length; i++) {
```

Comment:
good practice is create local `length` variable for `array.length` property in loop.